src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -329,7 +330,7 @@ public static void writeBundle(OutputStream outputStream, final List<? extends C
                 OutputStream textOut = maybeFilteredOut.map(OutputStream.class::cast).orElse(binaryOut);
                 OutputStreamSelector selector = new OutputStreamSelector(() -> binaryOut, () -> textOut);
                 IgnoreCloseOutputStream unfilteredOut = new IgnoreCloseOutputStream(binaryOut);
-                IgnoreCloseOutputStream filteredOut = new IgnoreCloseOutputStream(selector);
+                IgnoreCloseOutputStream filteredOut = new IgnoreCloseOutputStream(new FilterOutputStream(selector));
```

Comment:
@fbelzunc do we actually still need to wrap this up with a `FilterOutputStream` ? Seems to me that this is already done if needed (in case of anonymization) at https://github.com/jenkinsci/support-core-plugin/blob/master/src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java#L328. @MRamonLeon could confirm.

Reply:
I think you are right. I just made a commit to fix this. I also tried to generate an anonymous bundle and it worked without any problem after I perform the chage.